lib/manageiq/api/common/paginated_response.rb

```diff
@@ -22,7 +22,7 @@ def response
         private
 
         def metadata_hash
-          @metadata_hash ||= {"count" => count}
+          @metadata_hash ||= {"count" => count, "limit" => @limit, "offset" => @offset}
```

Comment:
The `@`'s shouldn't be needed since there are `attr_reader`s for both `limit` and `offset`